### DISCUSSION TITLE
Device naming

### DISCUSSION THREAD

**Discussion by Aqua1ung** [sezanzeb/input-remapper#504] [Q&A]
Here's an icky situation I am facing wrt presets: IR identifies my mouse as "Logitech M585/590." Yep, you've read that right: there's a slash in there, and that confuses the hell out of my system. As a result, quite often IR doesn't load the proper profile upon reboot, so I have to load it manually. Is there any way we can control the name of the devices as far as IR is concerned?

**Comment by jonasBoss** [comment]
I created issue #507 for this. 

A user can not rename devices in Input-Remapper and I don't think this will be possible any time soon.
